firefox-ios/Client/FeatureFlags/NimbusFlaggableFeature.swift

```diff
@@ -21,6 +21,7 @@ enum NimbusFeatureFlagID: String, CaseIterable {
     case feltPrivacyFeltDeletion
     case feltPrivacySimplifiedUI
     case firefoxSuggestFeature
+    case hntcontentFeedRefresh
```

Comment:
```suggestion
    case hntContentFeedRefresh
```
Camel case here and for all subsequent usages